### DISCUSSION TITLE
Welcome to moead-framework Discussions!

### DISCUSSION THREAD

**Discussion by geoffreyp** [moead-framework/framework#56] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by damlakomurcu** [comment]
Hello,
I try to implement an MOEAD optimization problem. First, I test with the code below to see that i am implementing in a correct way.
https://github.com/moead-framework/framework/blob/master/example_rmnk_moead.py 

But results are negative values like in the below. I think it is supposed to be between 0 and 1 like values in the instance file under ‘p tables’

array([[-0.59936725, -0.57269817],
       [-0.5901797 , -0.5875232 ],
       [-0.60340434, -0.57064142],
       [-0.52539518, -0.66643273],
       [-0.60579211, -0.56613425],
       [-0.56925268, -0.62854604],
       [-0.5957098 , -0.58578932],
       [-0.59560659, -0.58704566],..

Could you help me about this issue? It is very important for me.


What did i miss?


Thank you.

**Reply by geoffreyp** [reply]
Hello,

The RMNK problem is a maximization problem with results of the evaluation function between 0 and 1. The MOEAD framework supports only minimization problems, so in the implementation of the problem, I multiply the result by -1 (https://github.com/moead-framework/framework/blob/82e437678f5d54a13a71b8a97fa9a45cb6ec4685/moead_framework/problem/combinatorial/rmnk.py#L138C1-L138C36).

You can find more information here : https://moead-framework.github.io/framework/html/tuto.html#tuto-problem
